packages/core/strapi/lib/commands/transfer/export.js

```diff
@@ -25,7 +24,7 @@ module.exports = async (args) => {
   // To file
   const outputOptions = {
     file: {
-      path: args.output || getDefaultExportBackupName(),
+      path: args.output ?? getDefaultExportBackupName(),
```

Comment:
I think I preferred this with OR. We don't want to accept an empty string as a backup filename. What do you think?

Reply:
Good catch!